install.nsi

```diff
@@ -243,6 +248,35 @@
         SkipDockerInstall:
     FunctionEnd
 
+
+    Function CheckNvidiaDrivers
+        Var /GLOBAL DriverVersion
+
+        ; Try to read from the registry
+        SetRegView 64
+        ReadRegStr $DriverVersion HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{B2FE1952-0186-46C3-BAEC-A80AA35AC5B8}_Display.Driver" "DisplayVersion"
+
+        ${If} $DriverVersion == ""
```

Comment:
https://github.com/ClinicianFOCUS/clinicianfocus-installer/pull/24/commits/17965bc8817394bd169f69989cf7065b5750ea15

Added a disclaimer for now. It will need to be revisited once cpu is added